ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -1066,6 +1066,32 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-internal-registry
+- as: azure-ipi-fullyprivate-oidc-amd-f14
```

Comment:
This coverage is only supported from 4.17?

Reply:
Starting from version 4.14, it is supported. 
In version 4.16, Azure Workload Identity was renamed to Microsoft Entra Workload Identity (only the name changed, everything else remains the same). 
Should the CI jobs for versions 4.14, 4.15, and 4.16 be included in this PR?

Reply:
That depends on your plan, this PR is only targeted for 4.17, then you can remove 4.16 z-stream upgrade ci job .

Reply:
Got it. Once my debugging is sorted, I'm thinking of adding the CI for versions 4.14, 4.15, and 4.16 to this PR, so no need for a fresh PR :)